modules/3_helpernode/helpernode.tf

```diff
@@ -19,16 +19,19 @@
 ################################################################
 
 locals {
+    forwarders = tolist(split(";", var.dns_forwarders))
+
     helpernode_vars = {
         cluster_domain  = var.cluster_domain
         cluster_id      = var.cluster_id
         bastion_ip      = var.bastion_ip
-        forwarders      = var.dns_forwarders
         gateway_ip      = var.gateway_ip
         netmask         = cidrnetmask(var.cidr)
         broadcast       = cidrhost(var.cidr,-1)
         ipid            = cidrhost(var.cidr, 0)
         pool            = var.allocation_pools[0]
+        forwarder1      = trimspace(element(local.forwarders, 0))
+        forwarder2      = length(local.forwarders) > 1 ? trimspace(element(local.forwarders, 1)) : ""
```

Comment:
Can we take care of case when user want to use 3 name servers? I don't mind making the variable `var.dns_forwarders` as a list.
Secondly, It will be good to use `local.forwarders[0]` syntax instead of element function.
Thanks.

Reply:
Thanks for the comments.  I implemented both of your comments.